airflow-core/tests/unit/serialization/test_dag_serialization.py

```diff
@@ -386,6 +386,11 @@ def get_excluded_patterns() -> Generator[str, None, None]:
         if python_version in provider_info.get("excluded-python-versions"):
             provider_path = provider.replace(".", "/")
             yield f"providers/{provider_path}"
+    # CURRENT_PYTHON_VERSION = f"{sys.version_info.major}.{sys.version_info.minor}"
+    # if CURRENT_PYTHON_VERSION == "3.13":
+    # We should remove google when ray is fixed to work with Python 3.13
```

Comment:
Is this a TODO post merge (issue ticket?) or to be fixed before merge?

Reply:
This is after merge. Currently `ray` does not work with Python 3.13 - it tries to install numpy version that does not have binary wheels for 3.13 and fails while building.